### DISCUSSION TITLE
unable to load css

### DISCUSSION THREAD

**Discussion by sai-sandeep-k** [community/community#134646] [Programming Help]
### Select Topic Area

Question

### Body

I'm working on a portfolio and it looks great in localhost, but when I try to commit it to Github the CSS does't work.

**Comment by davevad93** [comment]
Hi @sai-sandeep-k ,

I checked your [portfolio](https://github.com/sai-sandeep-k/portfolio) and the issue is that you should rename `Portifolio.html` to `index.html`, otherwise GitHub Pages won't render the page correctly.

The paths of the `css` file and the images seem all correct in your html file, so you should fix your problem by renaming it to `index.html`,  however If you encounter any other problems let me know.

**Comment by Raykesh-Soneka** [comment]
Hi @sai-sandeep-k , 

try to redirect the index.html to Portifolio.html after creating a index.html
[https://www.w3schools.com/howto/howto_js_redirect_webpage.asp](url)
Or Maybe try this for the link to css
```css
<link rel="stylesheet" href="style.css">
```
if i answered your probleme don't forget to change
"Unanswered" to "answered"

Thank you
